### PR TITLE
Update webvtt/, webxr/, and xhr/ from monorail results

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See #798 for the methodology.

### CODE REVIEW

File: webusb/META.yml [hunk 1]
```diff
@@ -4,3 +4,7 @@ links:
       results:
         - test: usb.serviceworker.https.html
           status: FAIL
+    - product: chrome
+      url: https://crbug.com/166689
```
Comment (foolip) [inline webusb/META.yml:8]: The bug says "Reason for revert: Causing flaky crashes in external/wpt/webusb/idlharness.https.any.worker.html" so this is probably not relevant.

File: webvtt/rendering/cues-with-video/processing-model/META.yml [hunk 2]
```diff
@@ -3,3 +3,7 @@ links:
       url: https://bugzilla.mozilla.org/show_bug.cgi?id=1463844
       results:
         - test: audio_has_no_subtitles.html
+    - product: chrome
+      url: https://crbug.com/1012242
```
Comment (foolip) [inline webvtt/rendering/cues-with-video/processing-model/META.yml:7]: Looks right.

File: webvtt/rendering/cues-with-video/processing-model/selectors/cue_function/META.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/466072
```
Comment (foolip) [inline webvtt/rendering/cues-with-video/processing-model/selectors/cue_function/META.yml:3]: Looks right.

File: webxr/dom-overlay/META.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1101193
```
Comment (foolip) [inline webxr/dom-overlay/META.yml:3]: This bug added the test, so seems relevant to link.

File: xhr/META.yml [hunk 5]
```diff
@@ -53,3 +53,7 @@ links:
       results:
         - test: response-json.htm
           status: FAIL
+    - product: chrome
+      url: https://crbug.com/881180
```
Comment (foolip) [inline xhr/META.yml:57]: Looks right!

**Review by foolip — COMMENTED** [review]
This batch was much better, just one bug that doesn't seem relevant.

File: webusb/META.yml (hunk 1 above)
Comment (stephenmcgruer) [reply webusb/META.yml]: Reverted.

**Pull request merged**